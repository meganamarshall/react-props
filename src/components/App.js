import React from 'react';
import Color from './Color';

export default function App() {
  const name = 'blue';
  const rgb = {
    red: 0,
    green: 0,
    blue: 255
  };

  return <Color name={name} rgb={rgb} />;
}

