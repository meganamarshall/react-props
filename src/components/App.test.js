import React from 'react';
import App from './App';
import { shallow } from 'enzyme';

describe('App component', () => {
  it('matches the app snapshot', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
