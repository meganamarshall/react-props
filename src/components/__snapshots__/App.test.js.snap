// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders app 1`] = `<ColorPicker />`;
